wgpu/gui/auto.py

```diff
@@ -27,9 +28,35 @@ def is_jupyter():
 else:
     try:
         from .glfw import WgpuCanvas, run, call_later  # noqa
-    except ImportError as err:
-        msg = str(err)
-        msg += "\n\n  Install glfw using e.g. ``pip install -U glfw``."
-        if sys.platform.startswith("linux"):
-            msg += "\n  You may also need to run the equivalent of ``apt install libglfw3``."
-        raise ImportError(msg) from None
+    except ImportError as glfw_err:
+        try:
+            for libname in ("PySide6", "PyQt6", "PySide2", "PyQt5", "PySide", "PyQt4"):
```

Comment:
I think we can drop PyQt4 and PySide because afaik they are not supported on py36. Or at least, I could not get them to install on CI the other day.

Reply:
Yes! I'll update this and the qt module as well.